lib/vagrant-lxc/action/private_networks.rb

```diff
@@ -26,16 +26,19 @@ def configure_private_networks(env)
             next if type.to_sym != :private_network
 
             container_name = env[:machine].provider.driver.container_name
+            address_type   = config[:type]
             ip             = config[:ip]
             bridge_ip      = config.fetch(:lxc__bridge_ip) { build_bridge_ip(ip) }
             bridge         = config.fetch(:lxc__bridge_name)
```

Comment:
Is there a reason for the double underscore ("lxc__bridge" vs "lxc_bridge") in these variable names? That tripped me up initially.